### ISSUE TITLE
Significant lag after some time

### ISSUE THREAD

**Issue by OrigamingWasTaken** [opa334/Dopamine#602]
Hello, for the past months I've noticed that after a certain amount time (seems to vary so I would not be able to say how long), all the actions on my phone (iPhone 13 Mini, iOS 16.0) seems to be very delayed (3-10sec).

I have tried disabling all tweaks using iCleaner Pro, but the problem won't go away. (This problem started day 1 of me jailbreakin)

When unjailbroken, I have no issues.
It almost seems like a memory-leak, and the only thing I found to fix this issue was respringing.

Sorry if this issue is not well-made, I don't report bugs very often and I'm quit inexperienced.

**Comment by opa334** [comment]
Doesn't disabling tweaks in iCleaner Pro inevitably respring? Or do you mean you disabled everything and then kept it disabled but it happened again the next day? What happens if you disable tweak injection in the Dopamine app outright?

**Comment by OrigamingWasTaken** [comment]
Yes sorry I forgot to mention that a respring is only a temporary solution. It comes back after some time (so I'll leave my phone somewhere for like 1 hour, and when I need it urgently everything takes 10 seconds to load)

It's really random so I don't remember what happened when disabling from Dopamine. I'll try that today.

**Comment by hmj1314** [comment]
I had same issue, after JB, when the battery temp is high, the lightness will be down even adjust to maximum, and all the apps variable stalling, after i clean JB, it comes good.

**Comment by OrigamingWasTaken** [comment]
> I had same issue, after JB, when the battery temp is high, the lightness will be down even adjust to maximum, and all the apps variable stalling, after i clean JB, it comes good.

Did you find a fix? Or do you still have it?

**Comment by hmj1314** [comment]
> > I had same issue, after JB, when the battery temp is high, the lightness will be down even adjust to maximum, and all the apps variable stalling, after i clean JB, it comes good.
> 
> Did you find a fix? Or do you still have it?

 Actually I only installed 2 deb, but still have this problem, I keep no JB right now. it's come good. as the user, I don't know why, but thanks for all the efforts @opa334

**Comment by hmj1314** [comment]
maybe i am wrong, this days still have the same issue without JB.

**Comment by OrigamingWasTaken** [comment]
For me it's only when jailbroken

**Comment by hmj1314** [comment]
I had restored device yesterday and no jailbroken,  let me try if this issue will be repeated.

**Comment by hmj1314** [comment]
> I had restored device yesterday and no jailbroken, let me try if this issue will be repeated.

still will have this issue, no linkage to jailbroken, my phone is iphone 13 pro max, IOS 16.5

**Comment by opa334** [comment]
> > I had restored device yesterday and no jailbroken, let me try if this issue will be repeated.
> 
> still will have this issue, no linkage to jailbroken, my phone is iphone 13 pro max, IOS 16.5

So you mean... The issue happens on stock iOS aswell?

**Comment by OrigamingWasTaken** [comment]
In my case, it doesn't happen in stock iOS. I went 2 days without a single issue, but while jailbroken it only lasted something like 2 hours.

**Comment by hmj1314** [comment]
> > > I had restored device yesterday and no jailbroken, let me try if this issue will be repeated.
> > 
> > 
> > still will have this issue, no linkage to jailbroken, my phone is iphone 13 pro max, IOS 16.5
> 
> So you mean... The issue happens on stock iOS aswell?

Yes, I think so.

**Comment by OrigamingWasTaken** [comment]
Is there a way maybe I could find the culprit here? I tried looking at CocoaTop to see if there was a process using a lot of ram but I don't think it's the case.

**Comment by OrigamingWasTaken** [comment]
Bump

**Comment by luhgarlicbread** [comment]
Yea I’ve experienced this before as welll.

**Comment by OrigamingWasTaken** [comment]
I've now done some tests and this issue appear to be more present when using low battery mode, and when receiving notifications. When in this "state", my phones heats a lot and when looking at the processes in CocoaTop, the springboard seems to be using 90% of the CPU.

**Comment by luhgarlicbread** [comment]
> I've now done some tests and this issue appear to be more present when using low battery mode, and when receiving notifications. When in this "state", my phones heats a lot and when looking at the processes in CocoaTop, the springboard seems to be using 90% of the CPU.

sheeeesh

**Comment by OrigamingWasTaken** [comment]
> sheeeesh

Are you going to help?
